rm-datasource/src/main/java/io/seata/rm/datasource/exec/SelectForUpdateExecutor.java

```diff
@@ -95,17 +95,8 @@ public T doExecute(Object... args) throws Throwable {
                     if (StringUtils.isNullOrEmpty(lockKeys)) {
                         break;
                     }
-
-                    if (RootContext.inGlobalTransaction()) {
-                        //do as usual
-                        statementProxy.getConnectionProxy().checkLock(lockKeys);
-                    } else if (RootContext.requireGlobalLock()) {
-                        //check lock key before commit just like DML to avoid reentrant lock problem(no xid thus can
-                        // not reentrant)
-                        statementProxy.getConnectionProxy().appendLockKey(lockKeys);
-                    } else {
-                        throw new RuntimeException("Unknown situation!");
-                    }
```

Comment:
According to the specification, if the **RootContext** class has been deleted, the imported package needs to be deleted together, otherwise it cannot pass the ci check